industry_benchmarks/utils/experimental/plan_rbfe_network_bespoke.py

```diff
@@ -0,0 +1,329 @@
+)
+@click.option(
+    '--pdb',
+    type=click.Path(dir_okay=False, file_okay=True, path_type=pathlib.Path),
+    required=True,
+    help="Path to the prepared PDB file of the protein",
+)
+@click.option(
+    '--cofactors',
+    type=click.Path(dir_okay=False, file_okay=True, path_type=pathlib.Path),
+    default=None,
+    help="Path to the prepared cofactors SDF file (optional)",
+)
+@click.option(
+    '--output',
+    type=click.Path(dir_okay=True, file_okay=False, path_type=pathlib.Path),
+    default=pathlib.Path('alchemicalNetwork'),
+    help="Directory name in which to store the transformation json files",
+)
+@click.option(
+    '--force-field',
+    type=click.Path(dir_okay=False, file_okay=True, path_type=pathlib.Path),
+    help="The path to a SMIRNOFF style force field which should be used in the protocol.",
+    default=None
+)
+def run_inputs(ligands, pdb, cofactors, output, force_field):
+    """
+    Generate run json files for RBFE calculations
+
+    Parameters
+    ----------
+    ligands : pathlib.Path
+      A Path to a ligands SDF.
+    pdb : pathlib.Path
+      A Path to a protein PDB file.
+    cofactors : Optional[pathlib.Path]
+      A Path to an SDF file containing the system's cofactors.
+    output: pathlib.Path
+      A Path to a directory where the transformation json files
+      and ligand network graphml file will be stored into.
+    force_field: Optional[pathlib.Path]
+        The optional path to the local SMIRNOFF style force field which should be used in the protocol.
+
+    """
+    # Create the output directory -- default to alchemicalNetwork, fail if it exists
+    output.mkdir(exist_ok=False, parents=True)
+
+    # Create the small molecule components of the ligands
+    rdmols = [mol for mol in Chem.SDMolSupplier(str(ligands), removeHs=False)]
+    smcs = [openfe.SmallMoleculeComponent.from_rdkit(mol) for mol in rdmols]
+    # Generate the partial charges
+    logger.info("Generating partial charges for ligands")
+    smcs = [gen_charges(smc) for smc in smcs]
+
+    # Create ligand network
+    ligand_network = gen_ligand_network(smcs)
+
+    # Store the ligand network as a graphml file
+    with open(output / "ligand_network.graphml", mode='w') as f:
+        f.write(ligand_network.to_graphml())
+
+    # Create the solvent and protein components
+    solv = openfe.SolventComponent()
+    prot = openfe.ProteinComponent.from_pdb_file(str(pdb))
+
+    # If we have cofactors, load them in and assign partial charges
+    if cofactors is not None:
+        cofactors_smc = [gen_charges(openfe.SmallMoleculeComponent(m))
+                         for m in Chem.SDMolSupplier(str(cofactors), removeHs=False)]
+
+    # if we have a local force field extract the string version
+    if force_field is not None:
+        force_field_str = get_force_field(force_field=force_field)
+    else:
+        # hard code the default force field
+        force_field_str = "openff-2.2.0"
```

Comment:
Just a rough thought here that we might want to feed a different openff version if the industry partner wants to run benchmarks that match the other bespokefit benchmarks. I'm tempted to say "let's fix it after merge", but I'll leave it up to you to decide if we should wait to merge or not.

Reply:
Interesting we could change this to accept any local file or installed force field if they need that functionality, for now, I would say lets do the simple case of local files only and add the other case when needed.